d/DefinitelyTyped/build_info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "maintainer":"Shubham.Garud@ibm.com",
+    "package_name":"DefinitelyTyped",
+    "github_url":"https://github.com/DefinitelyTyped/DefinitelyTyped",
+    "package_dir":"d/DefinitelyTyped/",
+    "version":"master", 
+    "default_branch":"master",
+    "build_script":"DefinitelyTyped_ubi_8.7.sh",
+    "use_non_root_user":false,
+    "docker_build":false,
+    "validate_build_script":true,
+    "use_non_root_user":false
```

Comment:
this is duplicate entry, can you remove this.

Reply:
Yes. Removed the entry.